recipes/skore-hub-project/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license file is included in the source distribution so you don't need to include this copy here.